<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>916</width>
    <height>535</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Projective Transformations Application</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="browseImage">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>440</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Browse</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelImage">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>0</y>
      <width>491</width>
      <height>381</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="selectCoordinates">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>440</y>
      <width>111</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Choose points</string>
    </property>
   </widget>
   <widget class="QComboBox" name="cbAlgorithmType">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>440</y>
      <width>221</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pbExecuteAlgorithm">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>440</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Execute</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelA">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>40</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>A(x, y):</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelB">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>60</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>B(x, y):</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelC">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>80</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>C(x, y):</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelD">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>100</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>D(x, y):</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelAvalue">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>40</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="labelBvalue">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>60</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="labelDvalue">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>100</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="labelCvalue">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>80</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>160</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Matrix:</string>
    </property>
   </widget>
   <widget class="QLabel" name="matrixRow1">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>180</y>
      <width>281</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="matrixRow2">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>200</y>
      <width>261</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="matrixRow3">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>220</y>
      <width>271</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>916</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
